relationship_server.py

```diff
@@ -11,6 +11,41 @@
 nlp = spacy.load("en_core_web_lg")
 
 logger = logging.getLogger(__name__)
+
+
+'''
+    Utliity method to convert keys and values in a dictionary to lowercase.
+'''
+def makeDictLowercase(d):
+    lowerCaseDict = dict()
+    for k in d.keys():
+        lowerCaseDict[k.lower()] = d[k].lower()
+    return lowerCaseDict
+
+'''
+    Utility to convert string to Title Case
+    Input: this is a sentence.
+    Output: This Is A Sentence.
+'''
+def toTitleCase(string):
+    return " ".join(w.capitalize() for w in string.split(" "))
+
+'''
+    Loading JSON that has alias / acronym : country name mapping.
+'''
+def loadCountryAcronymJson():
+    with urllib.request.urlopen("https://raw.githubusercontent.com/rohanrmallya/coronaIndia/master/data/countries_acronym_aliases_flattened.json") as url:
```

Comment:
```suggestion
def loadCountryAcronymJson(download_url:str ="https://raw.githubusercontent.com/rohanrmallya/coronaIndia/master/data/countries_acronym_aliases_flattened.json")->None:
    with urllib.request.urlopen(download_url) as url:
```

Discourage hard coding any URLs. That makes the code hard to read and understand.

Reply:
Considered and done via 67ed2c693b4eef7d6ea1abac5c460ac2f358f478